R/endpoints.R

```diff
@@ -108,7 +108,8 @@ endpoint_model_result <- function(queue) {
                        value = process_result(queue$result(id)))
     } else if (task_status == "ERROR") {
       result <- queue$result(id)
-      error_data <- structure(result$message, trace = result$trace)
+      trace <- c(sprintf("# %s", id), result$trace)
```

Comment:
This line is important - it adds the model run id to the trace so that we can get it out of the user session without dealing with devtools